extensions/amp-access/0.1/amp-access.js

```diff
@@ -794,14 +783,19 @@ export class AccessService {
       } else {
         this.loginAnalyticsEvent_(type, 'rejected');
       }
+      const exchangePromise = this.signIn_.postLoginResult(query) ||
+          Promise.resolve();
       if (success || !s) {
```

Comment:
`s` here is the hash string, so it's always a string. In this code it's processed explicitly to show that an empty string is interpreted as `success=yes` for the purposes of re-authorization, but not for other purposes.